var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -244,3 +260,73 @@ def patch(self):
             filter_file(
                 "$(CXX) $(CXXFLAGS)", "$(FC) $(FCFLAGS)", "export/fortran/Makefile", string=True
             )
+
+
+class CMakeBuilder(
+    # Declare autotools first, so that building the libint compiler is very
+    # to the old process
+    spack.build_systems.autotools.AutotoolsBuilder,
+    spack.build_systems.cmake.CMakeBuilder,
+):
+    phases = ("autoreconf", "configure", "build", "cmake", "cmake_build", "install")
```

Comment:
Could you perhaps add a comment here concerning what this "mixing" of autotools and cmake in the build phase does under the hood and why it's needed?